src/index.js

```diff
@@ -3,10 +3,15 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { Provider } from 'react-redux'
+import Store from './Store'
+
 
 ReactDOM.render(
   <React.StrictMode>
+   <Provider store={Store}>
     <App />
```

Comment:
usually the only responsiblit yhere is to load APP and app has the provider theme ... etc